src/plugins/intel_cpu/tests/functional/single_layer_tests/pad.cpp

```diff
@@ -54,23 +58,65 @@ class PadLayerCPUTest : public testing::WithParamInterface<PadLayerCPUTestParamS
     }
 
 protected:
+    void generate_inputs(const std::vector<ngraph::Shape>& targetInputStaticShapes) override {
+        std::vector<void*> inputValues = {padsBegin.data(), padsEnd.data(), &padValue};
+
+        inputs.clear();
+        const auto& funcInputs = function->inputs();
+        for (int i = 0; i < funcInputs.size(); ++i) {
+            const auto& funcInput = funcInputs[i];
+            ov::Tensor tensor;
+            if (i == 0) {
+                tensor = ov::test::utils::create_and_fill_tensor(funcInput.get_element_type(), targetInputStaticShapes[i], 10, 1, 1);
+            } else {
+                if (ov::is_scalar(funcInput.get_partial_shape()))
+                    tensor = ov::Tensor{funcInput.get_element_type(), ov::Shape{}, &padValue};
```

Comment:
I'd suggest to use `frendly_name` for `PadValue` input. Because `Pad` node may have scalar input shapes. [The example](https://github.com/openvinotoolkit/openvino/blob/e409f708e03cb7b88d2d6c2b732395d30a1fcb61/src/plugins/intel_cpu/tests/functional/single_layer_tests/grid_sample.cpp#L99)